Adding a blog post about Secure Boot (#500)
AlmaLinux/almalinux.org: content/blog/2024-04-08-secure-boot-with-almalinux.md

---
title: "Secure boot with AlmaLinux"
type: blog
author: 
 name: "Sofia Boldyreva"
 bio: "Technical Writer for AlmaLinux OS Project"
 image: /users/sofia-boldyreva.png

> date: 2024-04-08

Reviewer: ```suggestion
date: 2024-04-11
```

images:
  - /blog-images/2024-04-08-secure-boot.png
post: 
    title: "Secure boot with AlmaLinux"
    image: /blog-images/2024-04-08-secure-boot.png
---

> As the new AlmaLinux releases are coming shortly, we'd like to talk about one of the vital components of each system - its security. It's important to keep your system secure and one of the easiest ways that is rarely talked about is by enabling Secure Boot.

Reviewer: ```suggestion
The new AlmaLinux releases are coming shortly (expect our beta releases for 9.4 and 8.10 over the next couple of weeks!), and we'd like to talk about one of the vital components of each system - its security. It's important to keep your system secure and one of the easiest ways that is rarely talked about is by enabling Secure Boot.
```

> AlmaLinux OS has provided Secure Boot since its 8.4 release.

Reviewer: ```suggestion
AlmaLinux OS has provided Secure Boot since its [8.4 release](https://wiki.almalinux.org/release-notes/8.4.html).
```

### About Secure Boot 

> In case you are not yet familiar, Secure Boot is a security feature built to ensure that only trusted software can run during the boot process. This feature helps prevent malicious software or unauthorized code from being loaded at boot time. Secure Boot verifies the signature of the operating system during boot-up, providing additional protection against malware and unauthorized access.

Reviewer: ```suggestion
In case you are not yet familiar, [Secure Boot](https://en.wikipedia.org/wiki/UEFI#Secure_Boot) is a security feature built to ensure that only trusted software can run during the boot process. This feature helps prevent malicious software or unauthorized code from being loaded at boot time. Secure Boot verifies the signature of the operating system during boot-up, providing additional protection against malware and unauthorized access.
```
